_episodes/04-sharing-environments.md

```diff
@@ -135,27 +144,31 @@ originally created `machine-learning-env` yet from the output of the `conda env
 we see that these five packages result in an environment with roughly 80 dependencies!
 
 To export this list into an environment.yml file, you can use `--file` option to directly save the
-resulting YAML environment into a file.
+resulting YAML environment into a file. If the target for `--file` exists it will be over-written so make sure your
+filename is unique. So that we do not over-write `environment.yaml` we save the output to `machine-learning-env.yaml`
+instead and add it to
 
 ~~~
-$ conda env export --name machine-learning-env --file environment.yml
+$ conda env export --name machine-learning-env --file machine-learning-env.yml
+$ git init
+$ git add machine-learning-env.yml
+$ git commit -m "Adding machine-learning-env.yml config file."
 ~~~
 {: .language-bash}
 
-Make sure you do not have any other `environment.yml` file from before in the same directory when
-running the above command. If you do you can use an alternative destination filename, e.g. `--file
-machline-learning-env.yaml`.
 
 This exported environment file may not *consistently* produce environments that are reproducible
 across operating systems. The reason for this is, that it may include operating system specific low-level
 packages which cannot be used by other operating systems.
 
-If you need an environment file that can produce environments that are reproducibile across Mac OS, Windows,
+If you need an environment file that can produce environments that are reproducible across Mac OS, Windows,

```

Comment:
I think we need to unpack this and line 165. Maybe not in this PR.

Reply:
Agree, reading through it now after a gap its a bit brief and could do with more detail.